From dev-bounces@openvswitch.org Fri Apr 16 01:12:26 2010
Received: from SMTP.EU.CITRIX.COM (10.30.244.21) by
 LONPMAILMX02.citrite.net (10.30.224.163) with Microsoft SMTP Server id
 8.1.393.1; Fri, 16 Apr 2010 01:12:26 +0100
Received: from vps.nicira.com ([66.7.219.28])  by SMTP.EU.CITRIX.COM with
 ESMTP/TLS/DHE-RSA-AES256-SHA; 16 Apr 2010 00:12:25 +0000
Received: from localhost ([127.0.0.1]:52667 helo=vps.nicira.com)	by
 vps.nicira.com with esmtp (Exim 4.69)	(envelope-from
 <dev-bounces@openvswitch.org>)	id 1O2ZAA-0002yN-Ff; Thu, 15 Apr 2010
 20:12:02 -0400
Received: from 173-164-153-21-sfba.hfc.comcastbusiness.net
 ([173.164.153.21]:29223 helo=leadville.nicira.com)	by vps.nicira.com with
 esmtps (TLSv1:AES256-SHA:256) (Exim 4.69)	(envelope-from <blp@nicira.com>)
 id 1O2ZA7-0002wU-F6	for dev@openvswitch.org; Thu, 15 Apr 2010 20:11:59 -0400
Received: from hardrock.nicira.com ([172.16.0.20])	by leadville.nicira.com
 with esmtp (Exim 4.67)	(envelope-from <blp@nicira.com>)	id
 1O2ZA1-0007es-FY; Thu, 15 Apr 2010 17:11:53 -0700
Received: from blp by hardrock.nicira.com with local (Exim 4.67)
 (envelope-from <blp@hardrock.nicira.com>)	id 1O2ZAC-0006DE-EJ; Thu, 15 Apr
 2010 17:12:04 -0700
From: Ben Pfaff <blp@nicira.com>
To: "dev@openvswitch.org" <dev@openvswitch.org>
CC: Ben Pfaff <blp@nicira.com>
Sender: "dev-bounces@openvswitch.org" <dev-bounces@openvswitch.org>
Date: Fri, 16 Apr 2010 01:11:56 +0100
Subject: [ovs-dev] [PATCH 1/4] bridge: Don't log warnings when revalidating.
Thread-Topic: [ovs-dev] [PATCH 1/4] bridge: Don't log warnings when
 revalidating.
Thread-Index: Acrc+X6xt7Crpa/TS5mZucxFnU1AJA==
Message-ID: <1271376719-23848-1-git-send-email-blp@nicira.com>
List-Help: <mailto:dev-request@openvswitch.org?subject=help>
List-Subscribe:
 <http://openvswitch.org/mailman/listinfo/dev_openvswitch.org>,
 <mailto:dev-request@openvswitch.org?subject=subscribe>
List-Unsubscribe:
 <http://openvswitch.org/mailman/options/dev_openvswitch.org>,
 <mailto:dev-request@openvswitch.org?subject=unsubscribe>
Accept-Language: en-GB, en-US
Content-Language: en-US
X-MS-Exchange-Organization-AuthAs: Anonymous
X-MS-Exchange-Organization-AuthSource: LONPMAILMX02.citrite.net
X-MS-Has-Attach: 
X-Auto-Response-Suppress: All
X-MS-TNEF-Correlator: 
x-sbrs: 5.3
x-mesageid: 10468976
x-ironport-server: lonpip01.citrite.net
x-remote-ip: 66.7.219.28
x-policy: $ACCEPTED
x-ironport-av: E=Sophos;i="4.52,215,1270425600";   
 d="scan'208";a="10468976"
x-antiabuse: Sender Address Domain - openvswitch.org
x-source: 
x-source-args: 
x-source-dir: 
list-id: <dev_openvswitch.org.openvswitch.org>
errors-to: dev-bounces@openvswitch.org
list-post: <mailto:dev@openvswitch.org>
list-archive: <http://openvswitch.org/pipermail/dev_openvswitch.org>
x-beenthere: dev@openvswitch.org
x-mailman-version: 2.1.12.cp3
Content-Type: text/plain; charset="us-ascii"
MIME-Version: 1.0
X-Evolution-Source: imap://ianca@lonpmailmx01.citrite.net/
Content-Transfer-Encoding: 8bit

The rest of the warnings along this path follow this rule, but this one
didn't.  Make it consistent.
---
 vswitchd/bridge.c |   10 ++++++----
 1 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/vswitchd/bridge.c b/vswitchd/bridge.c
index 620689f..baa889b 100644
--- a/vswitchd/bridge.c
+++ b/vswitchd/bridge.c
@@ -2223,10 +2223,12 @@ process_flow(struct bridge *br, const flow_t *flow,
 
     /* Drop frames on ports reserved for mirroring. */
     if (in_port->is_mirror_output_port) {
-        static struct vlog_rate_limit rl = VLOG_RATE_LIMIT_INIT(1, 5);
-        VLOG_WARN_RL(&rl, "bridge %s: dropping packet received on port %s, "
-                     "which is reserved exclusively for mirroring",
-                     br->name, in_port->name);
+        if (packet) {
+            static struct vlog_rate_limit rl = VLOG_RATE_LIMIT_INIT(1, 5);
+            VLOG_WARN_RL(&rl, "bridge %s: dropping packet received on port "
+                         "%s, which is reserved exclusively for mirroring",
+                         br->name, in_port->name);
+        }
         goto done;
     }
 
-- 
1.6.6.1


_______________________________________________
dev mailing list
dev@openvswitch.org
http://openvswitch.org/mailman/listinfo/dev_openvswitch.org

